src/aave-v3/SupplyVaultBase.sol

```diff
@@ -39,15 +35,13 @@ abstract contract SupplyVaultBase is ISupplyVaultBase, ERC4626UpgradeableSafe, O
     /// @notice Thrown when the zero address is passed as input or is the recipient address when calling `transferRewards`.
     error ZeroAddress();
 
-    /// CONSTANTS AND IMMUTABLES ///
+    /// IMMUTABLES ///
 
     IMorpho public immutable morpho; // The main Morpho contract.
     ERC20 public immutable morphoToken; // The address of the Morpho Token.
 
     /// STORAGE ///
 
-    /// STORAGE ///
-
     address public poolToken; // The pool token corresponding to the market to supply to through this vault.
     address public recipient; // The recipient of the rewards that will redistribute them to vault's users.
```

Comment:
Why didn't you also put this one as immutable ?

Reply:
For the other contracts because these slots are already used, for Aave v3 we more easily change it why not